src/zap/zapimage.cpp

```diff
@@ -1547,7 +1547,7 @@ void ZapImage::OutputTables()
 
 #if defined(FEATURE_PAL)
     // PAL library requires native image sections to align to page bounaries.
-    SetFileAlignment(0x1000);
+    SetFileAlignment(0x10000);
```

Comment:
@jkotas I compared the file size of System.Private.CoreLib.ni.dll with a recent build without this change to a similar build with this change.

```
17444864 4KAligned/System.Private.CoreLib.ni.dll
17629184 64KAligned/System.Private.CoreLib.ni.dl
```

The file appears to grow about 1%.